Basic Select/Weather-Observation-Station-10.sql

```diff
@@ -1,8 +1,4 @@
-# Author: Thomas George Thomas
-select distinct(city) from station where
-city not like "%a" and
-city not like "%e" and
-city not like "%i" and
-city not like "%o" and
-city not like "%u";
+# Author: Ibrahim Giwa
```

Comment:
Hi,
Could you not delete the previous code and add your code below it

Reply:
Hi, I believe it requires your approval is why I can’t edit the previous code.

Reply:
The approval is for merging. Please could you make a new commit with the changes in your branch. You can also raise a new PR with the new code below the original code in the file